src/frontend/Cargo.toml

```diff
@@ -54,6 +55,7 @@ risingwave_common = { workspace = true }
 risingwave_common_service = { workspace = true }
 risingwave_connector = { workspace = true }
 risingwave_expr = { workspace = true }
+risingwave_expr_macro = { workspace = true }
```

Comment:
We can re-export these macros from the expr crate so that we don't need to depend on the macro crate. (like `#[tokio::main]`)

Reply:
No idea how to do that, we've already made `risingwave_expr::function` mod public.

Reply:
We can remove that empty module now to avoid name conflict. 🤣